app/calendar/calendar-page.tsx

```diff
@@ -0,0 +1,30 @@
+import useSWR from "swr";
+import HeatMap from "./calendar-heatmap";
+import EventCreator from "./event-creator";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+export default function CalendarPage({ username }) {
+  const now = new Date();
+
+  let startDate = new Date(now);
+  startDate.setMonth(startDate.getMonth() - 6);
+
+  let endDate = new Date(now);
+  endDate.setMonth(endDate.getMonth() + 6);
+
+  const { data, error, isLoading, mutate } = useSWR(
+    "http://127.0.0.1:8080/calendar/" + username,
```

Comment:
This was changed from `"http://127.0.0.1:8080/calendar/agusovatii"` to `"http://127.0.0.1:8080/calendar/" + username,`